### DISCUSSION TITLE
Feed rss on Daily.Dev

### DISCUSSION THREAD

**Discussion by Inerska** [dailydotdev/daily#871] [General]
Hey, I hope I'm not repeating a previously asked question, but I'm interested in creating a Discord webhook to continuously update my "news about technologies" channel with articles posted on daily.web.

I already contacted support about this, but it seems to only be possible for bookmarks.

Best regards,
Inerska.

**Comment by nimrodkra** [comment]
Hi @Inerska indeed the current implementation we offer is via [Shareable bookmarks](https://changelog.daily.dev/shareable-bookmarks-221441). Feel free to explore this option as it allows you to pick the articles that you want to be shared with your server.

**Reply by Inerska** [reply]
Do you think that a "global implementation" could be possible ?

Thanks!

**Comment by nimrodkra** [comment]
We have no plans to do such a thing at the moment.
